models/object_detection/model_templates/face-detection/readme.md

```diff
@@ -33,148 +33,88 @@ source venv/bin/activate
-         --val-ann-files ${VAL_ANN_FILE} \
-         --val-data-roots ${VAL_IMG_ROOT} \
-         --save-checkpoints-to ${WORK_DIR}/outputs
-      ```
-
-      Also you can use parameters such as `--epochs`, `--batch-size`, `--gpu-num`, `--base-learning-rate`, otherwise default values will be loaded from `${MODEL_TEMPLATE}`.
-
-   * If you would like to start **fine-tuning** from pre-trained weights use `--resume-from` parameter and value of `--epochs` have to exceed the value stored inside `${MODEL_TEMPLATE}` file, otherwise training will be ended immediately. Here we add `5` additional epochs.
-
-      ```bash
-      export ADD_EPOCHS=5
-      export EPOCHS_NUM=$((`cat ${MODEL_TEMPLATE} | grep epochs | tr -dc '0-9'` + ${ADD_EPOCHS}))
+To convert PyTorch\* model to the OpenVINO™ IR format run the `export.py` script:
 
-      python train.py \
-         --resume-from ${WORK_DIR}/snapshot.pth \
-         --train-ann-files ${TRAIN_ANN_FILE} \
-         --train-data-roots ${TRAIN_IMG_ROOT} \
-         --val-ann-files ${VAL_ANN_FILE} \
-         --val-data-roots ${VAL_IMG_ROOT} \
-         --save-checkpoints-to ${WORK_DIR}/outputs \
-         --epochs ${EPOCHS_NUM}
-      ```
+```bash
+python export.py \
+   --load-weights ${SNAPSHOT} \
+   --save-model-to export
+```
 
-### 7. Evaluation
+This produces model `model.xml` and weights `model.bin` in single-precision floating-point format
+(FP32). The obtained model expects **normalized image** in planar BGR format.
 
-Evaluation procedure allows us to get quality metrics values and complexity numbers such as number of parameters and FLOPs.
+### 6. Run demo with exported model.
 
-To compute MS-COCO metrics and save computed values to `${WORK_DIR}/metrics.yaml` run:
+You need to pass a path to `model.xml` and video device node (e.g. /dev/video0) of your web cam.
 
 ```bash
-python eval.py \
-   --load-weights ${WORK_DIR}/outputs/latest.pth \
-   --test-ann-files ${VAL_ANN_FILE} \
-   --test-data-roots ${VAL_IMG_ROOT} \
-   --save-metrics-to ${WORK_DIR}/metrics.yaml
+python $OPEN_MODEL_ZOO_DIR/demos/object_detection_demo/python/object_detection_demo.py \
+   -m export/model.xml \
+   -at ssd \
+   -i /dev/video0
 ```
 
-You can also save images with predicted bounding boxes using `--save-output-to` parameter.
-
-```bash
-python eval.py \
-   --load-weights ${WORK_DIR}/outputs/latest.pth \
-   --test-ann-files ${VAL_ANN_FILE} \
-   --test-data-roots ${VAL_IMG_ROOT} \
-   --save-metrics-to ${WORK_DIR}/metrics.yaml \
-   --save-output-to ${WORK_DIR}/output_images
-```
+### 7. Evaluation of exported model.
 
-If you have WiderFace dataset downloaded you also can specify `--wider-dir` parameter where `WIDER_val.zip` file is stored in order to compute official WiderFace metrics.
+Instead of passing `snapshot.pth` you need to pass path to `model.bin`.
 
 ```bash
 python eval.py \
-   --load-weights ${WORK_DIR}/outputs/latest.pth \
+   --load-weights export/model.bin \
```

Comment:
Is it done intentionally, that the weights are passed to evaluation script, but not the model?

Reply:
it was done intentionally because of `load-weights` parameter name, though it can take both .xml and .bin, basically an extension is ignored in this particular case, though it has been changed to ${SNAPSHOT} which is pytorch model.